Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -107,6 +109,29 @@ - (CGRect)caretRectForPosition:(UITextPosition *)position
   return [super caretRectForPosition:position];
 }
 
+#pragma mark - Fix for CJK Languages
+// Search for `GH-19339` to see all code affected by this fix.
+
+- (void)setAttributedText:(NSAttributedString *)attributedText {
+  [_attributesHolder setAttributedString:attributedText];
```

Comment:
I mean why not to fallback to `copyAttributesFrom` here, if we see that `super.attributedText.string === attributedText`?

Reply:
should be fixed now